### DISCUSSION TITLE
dockerfile build documentation

### DISCUSSION THREAD

**Discussion by RoachMJ** [tchiotludo/akhq#888] [Q&A] [answered]
I am attempting to build the dockerfile in the repo. I continue to see the following error on docker run 
```
Error: Could not find or load main class org.akhq.App
Caused by: java.lang.ClassNotFoundException: org.akhq.App
```
Not sure if this is a bug or if I am missing something any help would be greatly appreciated.

**Comment by tchiotludo** [comment, marked as answer]
Are you building the jar before ? In order to build the docker image you need to  : 

```bash 
./gradlew shadowJar
cp build/libs/*-all.jar docker/app/akhq.jar
docker build -t test .
```

**Reply by RoachMJ** [reply]
Thank you, missed the copy step, much appreciated.
